### ISSUE TITLE
实在是看不懂data_loader.py中的train,val,test划分，找个索引都找不出？

### ISSUE THREAD

**Issue by feifei05** [thuml/iTransformer#169]
hello，大家好，我有两个问题
1、我的样本总数是997，分割后训练集、验证集和测试集数量（506，6，104）不知道怎么计算出来的，因为我要绘制测试集对应的日期图像，在data_loader.py分别保存分割和的数据集和date索引，谁能讲讲怎么分割的？
2、可能是上面分割的验证集太少，导致Vali Loss: nan，谁能讲讲如何设置参数更合适？
我采用custom自定义的数据集，[date,32个特征,OT]，样本总数997，设置下面的参数
  --data custom \
  --features MS \
  --seq_len 96 \
  --pred_len 96 \
  --e_layers 2 \
  --enc_in 33 \
  --dec_in 33 \
  --c_out 1 \
  --des 'Exp' \
  --d_model 128 \
  --d_ff 128 \
  --train_epochs 10 \
  --do_predict \
  --inverse \
  --itr 1

终端输出结果
train 506
val 6
test 104
Epoch: 1 cost time: 0.6490731239318848
Epoch: 1, Steps: 15 | Train Loss: 1.0043847 Vali Loss: nan Test Loss: 0.0871152
Validation loss decreased (inf --> nan).  Saving model ...
Updating learning rate to 0.0001
Epoch: 2 cost time: 0.492495059967041
Epoch: 2, Steps: 15 | Train Loss: 0.8969887 Vali Loss: nan Test Loss: 0.0943198
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 5e-05

**Comment by xun6000** [comment]
我感觉这个数据集数量不对

**Comment by Feihustc** [comment]
在`./data_provider/data_loader.py`中有，继承自定义数据集的话，应该是7:2:1划分的
